app/components/StandingTable/index.tsx

```diff
@@ -28,9 +28,21 @@ export function StandingTable({ label, conference }: StandingTableProps) {
           </tr>
         </TableHead>
         <tbody>
-          {conference.map((team) => (
+          {conference.map((team, index) => (
             <tr key={team.name}>
-              <TableCell>{team.rank}</TableCell>
+              <TableCell>
+                <div
+                  className={` rounded-full ${
+                    index < 6
+                      ? 'bg-green-900'
+                      : index < 10
+                      ? 'bg-green-600'
+                      : ''
+                  }`}
```

Comment:
You can use the `classnames` package here:

```suggestion
                  className={cn('rounded-full', {
                    'bg-green-600': index <= 5,
                    'bg-sky-600': index >= 6 && index < 10,
                  })}
```

Reply:
What do you think about extracting this logic to give meaningful names to the logic, i.e:
```js
className={cn('rounded-full', {  
  'bg-green-600': isTopSixTeams(),
  'bg-sky-600': IsTopTenTeams(),
})}
```

Something like that.